src/bikeracks/templates/base.html

```diff
@@ -115,10 +115,39 @@
                       </div>
                       
                   </li>
+                  <li class="nav-item">
+                      <button class="btn btn-secondary" data-toggle="modal" data-target="#feedbackModal">Contact</button>
+                  </li>
                 
               </ul> 
           </div> <!-- /.navbar-collapse -->
       </nav> 
+      
+      <!-- feedback modal -->
+      <div id="feedbackModal" class="modal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <p>Please leave comments or suggestions.</p>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              
+              <form id="feedbackForm">
+                <div class="form-group">
+                  <label for="feedback"></label>
+                  <textarea name="feedback" class="form-control" id="feedback" rows="3"></textarea>
+                </div>
+                <button id="submitFeedback" type="submit" class="btn btn-primary">Submit</button>
```

Comment:
Either this shouldn't be type "submit" or you need to trap the event in your event handler and prevent it from bubbling up and causing the redirect to happen.

Reply:
I removed type submit and i also use e.preventDefault in the click handler.